python/sglang/srt/entrypoints/openai/serving_chat.py

```diff
@@ -651,16 +567,14 @@ async def _handle_non_streaming_request(
         except ValueError as e:
             return self.create_error_response(str(e))
 
+        # Tokenizer manager should always return a list for chat requests
```

Comment:
We can remove this comment. I checked that when `request.n==1`, ret will be a dictionary. When `request.n>1`, it will be a list.